scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -221,14 +229,26 @@ def net_start(args):
     netpath = args.netpath
     netname = xml_get_value(args.netpath, "name")
     print("defining network from {0}".format(netname))
-    networks = conn.listNetworks()
-    if netname not in networks:
-        print("defining network from {0}".format(netpath))
-        xml = readfile(netpath)
-        conn.networkDefineXML(xml)
-    print("starting {0} network".format(netname))
-    dom = conn.networkLookupByName(netname)
-    dom.create()
+
+    # Get the names of active and inactive network domains.
+    try:
```

Comment:
I have the feeling that this `try:` is mostly for the `net_dom.setAutostart()`.  This is the only part that change the status of libvirt and if `listAllNetworkds` or `networkLookupByName` fails, I am not sure that we want to continue with the code : X

Also if `net_start()` of  `setAutostart()` fails, I do not except to see the code still running.